src/widgets/Forms/AddExpenseForm/AddExpenseCalc.tsx

```diff
@@ -18,6 +18,8 @@ import { deleteCategory } from 'src/shared/api/categoryApi.ts';
 import { deletePaymentSource } from 'src/shared/api/paymentsSourceApi.ts';
 import useLoadCategories from 'src/entities/category/hooks/useLoadCategories.ts';
 import useLoadPaymentSources from 'src/entities/paymentSource/hooks/useLoadPaymentSources.ts';
+import { BsDot } from 'react-icons/bs';
```

Comment:
Mui already have [dot](https://mui.com/material-ui/material-icons/?query=dot&selected=FiberManualRecord) and [backspace](https://mui.com/material-ui/material-icons/?query=backspace&selected=Backspace) icons. Even [Keyboard Backspace](https://mui.com/material-ui/material-icons/?query=backspace&selected=KeyboardBackspace)

they can be resized with fontsize property.